test/internal/options.go

```diff
@@ -172,16 +194,40 @@ func (t *T) Prepare(ctx context.Context, k8sClient client.Client) error {
 		Expect(k8sClient.Create(ctx, profile)).To(Succeed())
 	}
 
+	if o.containerProfile != nil {
+		// Clone the placeholder profile
```

Comment:
Why do we need to clone it? Wouldn't it make more sense to specify the existing container name so you know what you're getting ("example-success") up front, in the test definition?

Reply:
Now I'm wondering why I didn't just use the placeholder profile for storage profiles too...

Reply:
I removed this and turned to using the examples.